src/scripts/monitoring/hotjar.js

```diff
@@ -0,0 +1,10 @@
+<script>
+    (function(h,o,t,j,a,r){
+        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+        h._hjSettings={hjid:771371,hjsv:6};
```

Comment:
You should add `hjid` as a setting in the theme - via `settings_schema.json`. It will allow us to have a different HJID on devci.